main.go

```diff
@@ -166,7 +176,20 @@ TimeoutLoop:
 			}
 			return shouldRetry, err
 		})
+
+		try.Do(func(attempt int) (bool, error) {
+			imagesPruneReport, err := cli.ImagesPrune(context.Background(), args)
```

Comment:
Should we prune? 

It feels like this is a bigger action than just cleaning up images that have been used by Testcontainers, since it will remove all untagged images on the system regardless of whether they belong to Testcontainers or not.

Reply:
1) old code
2) we pass `args` that includes a set of filters that limit it to resources created by the current session only

Reply:
apparently I forgot to pull and the diff was bigger that it was supposed to be, sorry!